python/lsst/obs/decam/decamCpIsr.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
I think the one exception we make to the "one import per line" is for __future__, since we generally want all three. @r-owen suggested always having that line in alphabetical order, to make for easier searching in the future.